packages/eds-core-react/src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -30,12 +30,12 @@ describe('Tooltip', () => {
   })
   it('Should pass a11y test', async () => {
     const { container } = render(
-      <Tooltip title="Tooltip" enterDelay={0}>
```

Comment:
Do we not just go back to testing an empty document here and not the tooltip element? (Not that axe can really check anything useful for tooltip anyway)
![bilde](https://user-images.githubusercontent.com/4499531/169037048-e224fd76-e930-4f79-9918-7916c8df3b9c.png)

Reply:
Added a 100ms after firevent in all tooltip tests. Validated by using screen debug to log dom elements.